### PR TITLE
Dashboards: add cluster health to the Overview dashboard

### LINKED ISSUE #3079: Add "Mimir / Overview" dashboard
I propose to build a “Mimir / Overview” dashboard. It would be the first stop for the community to look at Mimir health, showing key metrics for writes / reads / storage / backend. We discussed it during the last Mimir team weekly sync and the feedback was positive.

Ideas we shared:
- We should have a row for write and a row for read path. QPS should be based on gateway metrics if enabled in the mixin config, otherwise should be based on ingress component (e.g. distributors for write path, query-frontends for read path).
- Add dashboard and panel descriptions. Make it clear for new adopters.
- Answer the question "Is the ruler on fire?"
- Answer the question "Is Mimir scaled correctly?"
- Show firing alerts (we can use the “Alerts panel” and filter alert name by "Mimir")

### PR DESCRIPTION
**Pull request by pracucci**
#### What this PR does
As a follow up of #3122, in this PR I'm proposing to add a view over the cluster health. This view is composed by two panels:

- Status over time, measured as % of failing requests / operations. < 1% is green, >= 1% < 5% is yellow, >= 5% is red.
- Firing alerts

**How it looks like:**
Screenshot 2022-10-06 at 14 46 07

#### Which issue(s) this PR fixes or relates to

Fixes #3079

#### Checklist

- [ ] Tests updated
- [x] Documentation added
- [x] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: operations/mimir-mixin-compiled/dashboards/mimir-alertmanager.json [hunk 1]
```diff
@@ -598,7 +598,7 @@
                         "step": 10
                      },
                      {
-                        "expr": "sum(cluster_job_integration:cortex_alertmanager_notifications_failed_total:rate5m{cluster=~\"$cluster\", job=~\"($namespace)/((alertmanager|cortex|mimir|mimir-backend))\"})",
+                        "expr": "sum(cluster_job_integration:cortex_alertmanager_notifications_failed_total:rate5m{cluster=~\"$cluster\", job=~\"($namespace)/((alertmanager|cortex|mimir|mimir-backend))\"})\n",
```
Comment (pracucci) [inline operations/mimir-mixin-compiled/dashboards/mimir-alertmanager.json:601]: Note to reviewers: removed a newline.

File: operations/mimir-mixin/dashboards/dashboard-utils.libsonnet [hunk 2]
```diff
@@ -489,6 +492,69 @@ local utils = import 'mixin-utils/utils.libsonnet';
     type: 'text',
   } + options,
 
+  alertListPanel(title, nameFilter='', labelsFilter=''):: {
+    type: 'alertlist',
+    title: title,
+    options: {
+      maxItems: 100,
```
Comment (56quarters) [inline operations/mimir-mixin/dashboards/dashboard-utils.libsonnet:499]: This seems like a lot for an overview dashboard

**Review by 56quarters — APPROVED** [review]
Looks great!

File: operations/mimir-mixin/dashboards/dashboard-utils.libsonnet (hunk 2 above)
Comment (pracucci) [reply operations/mimir-mixin/dashboards/dashboard-utils.libsonnet:499]: Good point. My rationale was that we shouldn't "silently" hide alerts, so I thought about putting an high number here. I don't feel strong and can reduce it. WDYT?

Comment (56quarters) [reply operations/mimir-mixin/dashboards/dashboard-utils.libsonnet:499]: OK, fine to leave it as is then.

**Pull request merged**